hadoop-common-project/hadoop-kms/src/test/java/org/apache/hadoop/crypto/key/kms/server/TestKMSAudit.java

```diff
@@ -97,8 +98,6 @@ private String getAndResetLogOutput() {
   @Test
   @SuppressWarnings("checkstyle:linelength")
```

Comment:
I think `@SuppressWarnings("checkstyle:linelength")` can be removed.

Reply:
Thanks @aajisaka I added a new commit to fix remove the superwarnings.